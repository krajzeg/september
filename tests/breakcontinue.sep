i := 0;

print("The loop should print odd numbers up to 9.");
while (True) {
	i = i + 1;
	if (i >= 10) { break(); };
	if (i % 2 == 0) { continue(); };
	print(i);
};
print("break() exited the loop exited cleanly.");


