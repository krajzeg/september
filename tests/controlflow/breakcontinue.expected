The loop should print only odd numbers up to 9.
1
3
5
7
9
'break' exited the loop exited cleanly.
